lib/dal/src/func/authoring/save.rs

```diff
@@ -331,15 +336,16 @@ async fn save_attr_func_proto_arguments(
     arguments: Vec<AttributePrototypeArgumentBag>,
 ) -> FuncAuthoringResult<()> {
     let mut id_set = HashSet::new();
+
     for arg in &arguments {
```

Comment:
I go back and forth on handling this struct. We are probably due for a re-write that doesn't use `NONE` IDs at all, so I am holding off. Let's get feature parity first unless this becomes annoying... well, more than it already is.